feature/security/gnsi/acctz/AccountingAuthenFailMulti/README.md

```diff
@@ -0,0 +1,53 @@
+# ACCTZ-7.1 - gNSI.acctz.v1 (Accounting) Test Accounting Authentication Failure - Multi-transaction
+
+## Summary
+Test Accounting for authentication failures of multi-transaction logins
+
+## Procedure
+
+- Record the time T0 for use later in this test
+- For each of the supported RecordResponse.service_request.service_type that is not authenticated per-transaction:
+	- Connect to the DUT, recording the local and remote IP addresses and port numbers,
+	- Provide invalid user credentials (including an empty username, unconfigured username, empty password, invalid password, wrong SSH key/certificate, etc),
+	- disconnect
+- Establish gNSI connection to the DUT.
+- Call gnsi.acctz.v1.Acctz.RecordSubscribe with RecordRequest.timestamp = T0
+- Verify that accurate accounting records are returned for the commands/RPCs authentication failures.
+- If start/stop accounting is supported, each authentication failure should have an accompanying LOGIN accounting record.
+- For each RecordResponse correlated to each connection made above, check/confirm that:
+	- session_info. :
+		- .{layer4_proto, local_address, local_port, remote_address, remote_port}, ip_proto must match those recorded earlier
+		- channel_id = 0 for ssh and grpc.
+		- .tty must be populated and correct, if applicable to the platform & access method, else omitted
+		- .status must equal LOGIN:
+			- .authen.type must equal the authentication method used.
+			- .authen.status must equal FAIL or ERROR, and cause should be populated.
```

Comment:
Hrm, I had not thought of it this way.  Assuming that the DUT is working properly, it should be FAIL.  If it were a test of a failing backend auth mechanism (eg: tacacs unavailable), then it would fail.  so, I think that you are correct and that there needs to be a test for ERROR.